### DISCUSSION TITLE
Issue 2 発刊に向けた議論

### DISCUSSION THREAD

**Discussion by shotaro8yagi** [project-theory-probe/ptp-voyage#3] [General]
Issue 2 発刊は11月末を予定しています。そこに向けた議論はこちらで！

**Comment by kikuchiharuma** [comment]
Issue 2 の作業はこちらで進めましょう。
https://github.com/project-theory-probe/ptp-voyage/pull/5

**Comment by kikuchiharuma** [comment]
とりあえず連載の枠だけ用意した状態にしてあります。
@shotaro8yagi パスしていただいた素材は随時ここに反映していきます！
https://deploy-preview-5--soft-custard-acf2d5.netlify.app/

**Reply by kikuchiharuma** [reply]
@KokoroKagawa 「このごろの Project Sprint」も同様です。今月もよろしくお願いします！

**Comment by kikuchiharuma** [comment]
ハッカー文化の伝道師
2回目、エリック・レイモンドの原稿追加しました
https://deploy-preview-5--soft-custard-acf2d5.netlify.app/journal/2/#hackerculture

**Comment by KokoroKagawa** [comment]
@kikuchiharuma 
第二回分のメソッド原稿です。よろしくお願いいたします～。
https://docs.google.com/document/d/1Hb_HJsrRw88223ip3ENZqsclduGF3ftR6IECJ4SfWpY/edit?usp=sharing

**Reply by kikuchiharuma** [reply]
ありがとうございます！

**Reply by kikuchiharuma** [reply]
反映しました！こちらからご確認ください
https://deploy-preview-5--soft-custard-acf2d5.netlify.app/journal/2/#recentprojectsprint

**Comment by shotaro8yagi** [comment]
連載：
https://docs.google.com/document/d/1N2FtbWLkQorVKGlUzIducxpLKCgOBX5OkBCU9yCySQ8/edit
今月の一冊：
https://docs.google.com/document/d/1qkBZN0RTAanQSg4RiiF1pfGIwdjy6Ss0e4Rk7jwC0PE/edit
宇宙と四コマは少々お待ちください！

**Reply by kikuchiharuma** [reply]
ありがとうございます！

**Reply by kikuchiharuma** [reply]
反映しました！こちらからご確認ください
https://deploy-preview-5--soft-custard-acf2d5.netlify.app/journal/2/#selforganizingprojects

約物 (カッコ) やエクスクラメーションマーク、数字などを半角に改めています。
またタイトルは第1回に合わせて、～ホモ・プロジェクティカス～ ではなく「ホモ・プロジェクティカス」としています。

**Reply by shotaro8yagi** [reply]
ありがとうございます！２つとも最終化しました！よろしくお願い致します！！　@kikuchiharuma

**Reply by kikuchiharuma** [reply]
ありがとうございます！

**Reply by kikuchiharuma** [reply]
反映しました！
https://deploy-preview-5--soft-custard-acf2d5.netlify.app/journal/2/

**Reply by shotaro8yagi** [reply]
https://www.hup.harvard.edu/books/9780674251557
「ウィルフリド・セラーズが展開した認識論」という文字にリンクいただけたら嬉しいです！

**Reply by kikuchiharuma** [reply]
反映済みです！

**Comment by shotaro8yagi** [comment]
ぼくプロジェクトおじさん2
旅行計画
![ぼくプロジェクトおじさん (2)](https://github.com/project-theory-probe/ptp-voyage/assets/49149959/e623d73c-917d-4e73-8670-42dd6ae0e47a)

**Reply by kikuchiharuma** [reply]
反映しました！
https://deploy-preview-5--soft-custard-acf2d5.netlify.app/journal/2/#bokuprojectojisan

**Comment by kikuchiharuma** [comment]
@shotaro8yagi @KokoroKagawa Issue 2 リリースされました！おつかれさまでした！ https://ptp.voyage/journal/2/

**Reply by shotaro8yagi** [reply]
有難うございました！！！お疲れ様です👋
